ledger/src/blockstore_db.rs

```diff
@@ -103,6 +104,8 @@ const BLOCK_HEIGHT_CF: &str = "block_height";
 const PROGRAM_COSTS_CF: &str = "program_costs";
 /// Column family for optimistic slots
 const OPTIMISTIC_SLOTS_CF: &str = "optimistic_slots";
+/// Column family for merkle roots
+const MERKLE_ROOT_CF: &str = "merkle_root_meta";
```

Comment:
Can we name this `MERKLE_ROOT_META_CF` for consistency?
One side has `_meta` and the other side doesn't.